packages.json

```diff
@@ -5264,5 +5264,22 @@
     "description": "ISAAC PRNG implementation on Nim",
     "license": "MIT",
     "web": "https://github.com/pragmagic/isaac/"
+  },
+  {
+    "name": "SDF.nim",
```

Comment:
I made the change but just a small question. Would `SDF.nim.nimble` be an also acceptable package filename? Sorry.